pkgs/development/python-modules/pypandoc/default.nix

```diff
@@ -0,0 +1,35 @@
+{
+stdenv, buildPythonPackage, fetchPypi,
+pip, pandoc, glibcLocales,
+haskellPackages, texlive
```

Comment:
@FRidh How can I not use `haskellPacakges` and `texlive` here (for line 27)? reference to [this contex](https://github.com/NixOS/nixpkgs/pull/27379/files/024f77f924b19fcac4c6b4083782b2cefa474bba#r127500764)